backend/app/app/config/tools.yml

```diff
@@ -359,6 +361,8 @@ library:
       - name: output
         content: |-
           I'm not sure which action to take. Can you clarify your question such that it is easier to understand what action I should
+    additional:
+      human_message_only: false
```

Comment:
Doesn't do anything - this is the default

Reply:
Not sure if this is cleanest - shouldn't we tell users to define the chat history parsing logic strictly in the .py file rather than partly in yaml, using the functionality we've built in `ToolInputSchema`?